docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,681 @@
+# Policy On Namespace
+
+- Status: accepted
+
+## Context and Problem Statement
+
+Kuma 2.6.0 enable users to apply policies on Zone CP.
+This change unblocks us to introduce namespace support for policies.
+The CRDs for new policies are already namespace-scoped, but they can be applied only on the `kuma-system` namespace.
+
+## Decision Drivers
+
+* Kubernetes-native UX, all app-related resources should be applied in the app's namespace
+* Ability to use Kubernetes RBAC with Kuma policies
+* It's not sustainable to grant access to `kuma-system` namespace to everyone who works with Kuma policies
+* Service owners should have a way to affect Envoy configuration of their clients without applying policies to `kuma-system`
+
+## Considered Options
+
+* implicitly convert policy's namespace to `k8s.kuma.io/namespace` tag for top-level targetRef
+* adopt producer/consumer concept from [GAMMA](https://gateway-api.sigs.k8s.io/mesh/#gateway-api-for-service-mesh)
+
+## Decision Outcome
+
+- adopt producer/consumer concept from GAMMA
+
+## Pros and Cons of implicitly converting policy's namespace to `k8s.kuma.io/namespace` tag for top-level targetRef
+
+* Good, because it easy to implement
+* Good, because it works similar to zone-originated policies (we implicitly add `kuma.io/zone` tag for top-level targetRef)
+* Bad, because it doesn't remove necessity to create policies in `kuma-system` namespace
+
+## Pros and Cons of adopting producer/consumer concept from GAMMA
+
+* Good, because it's familiar to users who know GAMMA
+* Good, because creating policies in `kuma-system` becomes an edge use-case, the vast majority of use cases is covered by namespace-scoped policies
+* Bad, because it requires syncing policies across zones. 
```

Comment:
Is this actually Bad. I don't think so. I think if anything is bad here is that the logic of which policies can be synced or not can be a little complex